tests need updating:

```
FAIL: test_instantiate_all (testapps.test_pillowtop.tests.test_settings.PillowtopSettingsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/testapps/test_pillowtop/tests/test_settings.py", line 33, in test_instantiate_all
    self.assertEqual(expected_meta[pillow_config.name], _pillow_meta_from_config(pillow_config))
AssertionError: {u'es_type': u'hqdomain', u'name': u'DomainPillow', u'advertised_name': u'corehq [truncated]... != {'es_type': 'hqdomain', 'name': 'DomainPillow', 'advertised_name': 'corehq.pillo [truncated]...
- {u'advertised_name': u'corehq.pillows.domain.DomainPillow.test_hqdomains_20160308_1304.testhq',
?  -                   -                                                         ^    ^^
+ {'advertised_name': 'corehq.pillows.domain.DomainPillow.test_hqdomains_20160318_1339.testhq',
?                                                                              ^    ^^
-  u'change_feed_type': u'CouchChangeFeed',
?  -                    -
+  'change_feed_type': 'CouchChangeFeed',
-  u'checkpoint_id': u'corehq.pillows.domain.DomainPillow.test_hqdomains_20160308_1304.testhq',
?  -                 -                                                         ^    ^^
+  'checkpoint_id': 'corehq.pillows.domain.DomainPillow.test_hqdomains_20160318_1339.testhq',
?                                                                            ^    ^^
-  u'couch_filter': u'domain/domains_inclusive',
?  -                -
+  'couch_filter': 'domain/domains_inclusive',
-  u'couchdb_type': u'Database',
?  -                -
+  'couchdb_type': 'Database',
-  u'couchdb_uri': u'http://couch:5984/test_commcarehq__domains',
?  -               -
+  'couchdb_uri': 'http://couch:5984/test_commcarehq__domains',
-  u'document_class': u'Domain',
?  -                  -
+  'document_class': 'Domain',
-  u'es_alias': u'hqdomains',
?  -            -
+  'es_alias': 'hqdomains',
-  u'es_index': u'test_hqdomains_20160308_1304',
?  -            -                      ^    ^^
+  'es_index': 'test_hqdomains_20160318_1339',
?                                    ^    ^^
-  u'es_type': u'hqdomain',
?  -           -
+  'es_type': 'hqdomain',
-  u'extra_args': {},
?  -
+  'extra_args': {},
-  u'full_class_name': u'corehq.pillows.domain.DomainPillow',
?  -                   -
+  'full_class_name': 'corehq.pillows.domain.DomainPillow',
-  u'include_docs': True,
?  -
+  'include_docs': True,
-  u'name': u'DomainPillow',
?  -        -
+  'name': 'DomainPillow',
-  u'unique_id': u'test_hqdomains_20160308_1304'}
?  -             -                      ^    ^^
+  'unique_id': 'test_hqdomains_20160318_1339'}
?                                     ^    ^^
----------------------------------------------------------------------
Ran 13 tests in 6.020s
FAILED (failures=1)
```

--- a/testapps/test_pillowtop/tests/data/all-pillow-meta.json
+++ b/testapps/test_pillowtop/tests/data/all-pillow-meta.json
@@ -144,21 +144,21 @@
         "name": "DefaultChangeFeedPillow"
     },
     "DomainPillow": {
-        "advertised_name": "corehq.pillows.domain.DomainPillow.test_hqdomains_20160308_1304.testhq",
+        "advertised_name": "corehq.pillows.domain.DomainPillow.test_hqdomains_20160318_1339.testhq",
         "change_feed_type": "CouchChangeFeed",
-        "checkpoint_id": "corehq.pillows.domain.DomainPillow.test_hqdomains_20160308_1304.testhq",
+        "checkpoint_id": "corehq.pillows.domain.DomainPillow.test_hqdomains_20160318_1339.testhq",
         "couch_filter": "domain/domains_inclusive",
         "couchdb_type": "Database",
         "couchdb_uri": "http://{COUCH_SERVER_ROOT}/test_commcarehq__domains",
         "document_class": "Domain",
         "es_alias": "hqdomains",
-        "es_index": "test_hqdomains_20160308_1304",
+        "es_index": "test_hqdomains_20160318_1339",
         "es_type": "hqdomain",
         "extra_args": {},
         "full_class_name": "corehq.pillows.domain.DomainPillow",
         "include_docs": true,
         "name": "DomainPillow",
-        "unique_id": "test_hqdomains_20160308_1304"
+        "unique_id": "test_hqdomains_20160318_1339"
     },
     "FarmerRecordFluffPillow": {
         "advertised_name": "fluff.FarmerRecordFluffPillow.testhq",

